src/app/frontend/secret/detail/info.html

```diff
@@ -19,5 +19,10 @@
   <kd-info-card-section>
     <kd-object-meta-info-card object-meta="::$ctrl.secret.objectMeta">
     </kd-object-meta-info-card>
+    <kd-info-card-entry title="[[Type|Secret info details section type entry.]]">
+      <kd-middle-ellipsis display-string="{{::$ctrl.secret.type}}">
+      </kd-middle-ellipsis>
+      <div ng-hide="::$ctrl.secret.type">-</div>
```

Comment:
type Secret struct {
	metav1.TypeMeta `json:",inline"`
	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
	Data map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
	StringData map[string]string `json:"stringData,omitempty" protobuf:"bytes,4,rep,name=stringData"`
	Type SecretType `json:"type,omitempty" protobuf:"bytes,3,opt,name=type,casttype=SecretType"`
}

const MaxSecretSize = 1 * 1024 * 1024
type SecretType string

====
use type  not SecretType?